app/src/main/java/eu/kanade/tachiyomi/source/online/handlers/ApiMangaParser.kt

```diff
@@ -26,7 +26,7 @@ class ApiMangaParser(val lang: String) {
             val networkApiManga = Json.nonstrict.parse(ApiMangaSerializer.serializer(), jsonData)
             val networkManga = networkApiManga.manga
             manga.title = MdUtil.cleanString(networkManga.title)
-            manga.thumbnail_url = MdUtil.cdnUrl + networkManga.cover_url
+            manga.thumbnail_url = MdUtil.cdnUrl + MdUtil.removeTimeParamUrl(networkManga.cover_url)
```

Comment:
why we removing time parameter?

Reply:
The cover cache is based on a md5 hash of the thmbnailurl. Since we force update, we should just always use the cached file unless we have triggered the force refresh. I am not sure when this time param changes, but that was my logic to reduce the number of network calls.